ui_framework/src/components/gallery/gallery_item/gallery_item.test.js

```diff
@@ -1,12 +1,32 @@
 import React from 'react';
+import sinon from 'sinon';
 import { render } from 'enzyme';
 import { requiredProps } from '../../../test/required_props';
 
 import {
   KuiGalleryItem,
 } from './gallery_item';
 
-test('renders KuiGalleryItem href', () => {
+test('renders KuiGalleryItem with href', () => {
   const component = <KuiGalleryItem href="#" {...requiredProps}>children</KuiGalleryItem>;
   expect(render(component)).toMatchSnapshot();
 });
+
+test('renders KuiGalleryItem with onClick', () => {
+  const component = <KuiGalleryItem onClick={() => {}} {...requiredProps}>children</KuiGalleryItem>;
```

Comment:
Instead of doing a snapshot comparison, we should test that the callback is called when the element is clicked, similar to how our `KuiButton` tests work:

```jsx

    describe('onClick', () => {
      test(`isn't called upon instantiation`, () => {
        const onClickHandler = sinon.stub();

        shallow(
          <KuiButton onClick={onClickHandler} />
        );

        sinon.assert.notCalled(onClickHandler);
      });

      test('is called when the button is clicked', () => {
        const onClickHandler = sinon.stub();

        const $button = shallow(
          <KuiButton onClick={onClickHandler} />
        );

        $button.simulate('click');

        sinon.assert.calledOnce(onClickHandler);
      });
    });
```

Could you also add an example of a KuiGalleryItem using `onClick` in the doc site and verify that it gets applied and called when you click?

Reply:
Done.